cmd/ipfs/init.go

```diff
@@ -142,6 +148,23 @@ func addTheWelcomeFile(conf *config.Config) error {
 	return nil
 }
 
+func initializeIpnsKeyspace(conf *config.Config) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	nd, err := core.NewIPFSNode(ctx, core.Offline(conf))
+	if err != nil {
+		return err
+	}
+	defer nd.Close()
```

Comment:
we should probably be initializing this node outside of this function-- i.e. inside of `doInit`. dont think users who were to implement a similar thing (i.e. that needs an actual node) would find it buried in here. nbd though